san/tests/test_sanbase_integration.py

```diff
@@ -73,6 +91,77 @@ def test_erc20_projects():
     assert len(result) > 0
     assert len(result[result.slug == "bitcoin"]) == 0
 
+@attr('integration')
+def test_ordinary_ethereum_queries():
+    for query in METRICS_USING_ETHEREUM:
+        result = ordinary_function_maker(query, "ethereum")
+
+        assert len(result.index) >= 1
+
+@attr('integration')
+def test_token_age_consumed():
+    result = ordinary_function_maker('token_age_consumed')
+
+    for row in result['tokenAgeConsumed']:
```

Comment:
The assertions can go into `ordinary_function_maker` it can be renamed to `test_ordinary_function` and the call will be as simple as:

```python
def test_token_age_consumed():
    test_ordinary_function('token_age_consumed', 'tokenAgeConsumed')
```

Reply:
You're right, I've moved and renamed the function accordingly